index.html

```diff
@@ -149,142 +167,206 @@ <h2>Multi-Post Stories</h2>
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Languages</span>
-          <span>
-            <img src="images/arrow_bottom.png" alt="arrow_bottom">
-          </span>
-        </h2>
-        <ul class="languages">
-          <li>
-            <img src="images/Ellipse1.jpg" alt="JavaScript" />
-            <a href="#">JavaScript</a>
-          </li>
-          <li>
-            <img src="images/Ellipse1(1).jpg" alt="HTML" />
-            <a href="#">HTML</a>
-          </li>
-          <li>
-            <img src="images/Ellipse1(2).jpg" alt="CSS" />
-            <a href="#">CSS</a>
-          </li>
-        </ul>
-      </li>
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Frameworks</span>
-          <span>
-            <img src="images/arrow_right.png" alt="arrow_right">
-          </span>
-        </h2>
-      </li>
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Skills</span>
-          <span>
-            <img src="images/arrow_right.png" alt="arrow_right">
-          </span>
-        </h2>
-      </li>
-    </ul>
-  </section>
-
-  <section class="contact-form">
-    <div class="title-p">
-      <h1 class="contact-me">Contact me</h1>
-      <p class=p2>If you have an application you are interested in developing,
-         a feature that you need built or a project that needs coding.
-         I’d love to help with it </p>
-    </div>
-    <form  class="form" action="https://formspree.io/f/xbjwakbe" method="post">
-      <ul>
-        <li class="text">
-          <input required type="text" id="name" name="user_name" maxlength="30" placeholder="Write your name"/>
+    </li>
+    <li class="knowledge">
+      <h2 class="kn_1">
+        <span>Skills</span>
+        <span>
+          <img src="images/arrow_right.png" alt="arrow_right">
+        </span>
+      </h2>
+      <ul class="frameworks">
+        <li>
+          <img src="images/Ellipse_1.png" alt="JavaScript" />
+          <a href="">JavaScript</a>
         </li>
-        <li class="text">
-          <input required type="email" id="mail" name="user_email" placeholder="Write your email" />
+        <li>
+          <img src="images/Ellipse_2.png" alt="HTML" />
+          <a href="">HTML</a>
         </li>
-        <li class="text">
-          <textarea required id="msg" name="user_message" maxlength="500" placeholder="Write your message here"></textarea>
+        <li>
+         <img src="images/Ellipse_3.png" alt="CSS" />
+          <a href="">CSS</a>
         </li>
-      </ul> 
-      <input class="butt" type="submit" value="Get It Touch">
+      </ul>
+    </li>
+  </ul>
+</section>
+  <footer id="contact" class="contact">
+    <div class="contact-contact-container">
+    <div class="container-contact">
+      <h2 class="first-text-contact"> Contact me</h2>
+      <p class="second-text-contact">If you have an application you are interested in developing, a feature that you need built or a
```

Comment:
- [ ] The background image of this form looks blurry, I recommend using a better quality image for it, I downloaded it for you first one is svg, second png feel free to use them: 
![contact form background shapes](https://user-images.githubusercontent.com/77038610/161226361-32116161-7dc4-498c-b8e0-c906ce7ee244.svg)
<img width="1291" alt="contact form background shapes@2x" src="https://user-images.githubusercontent.com/77038610/161226364-da118b3f-7746-40ee-bdd3-0d524e413067.png">